[92m[11:48:26 AM][0m: OS: linux2, 64 cores, ~ 528 GB RAM. Python: 2.7.5
[92m[11:48:27 AM][0m: Running funannotate v0.6.3
[92m[11:48:27 AM][0m: No NCBI SBT file given, will use default, however if you plan to submit to NCBI, create one and pass it here '--sbt'
[92m[11:48:27 AM][0m: 28,212 protein records loaded
[92m[11:48:27 AM][0m: Running HMMer search of PFAM domains
-------------------------------------------------------
Traceback (most recent call last):
  File "/bigdata/bioinfo/pkgadmin/opt/linux/centos/7.x/x86_64/pkgs/funannotate/git-live/bin/funannotate-functional.py", line 278, in <module>
    lib.PFAMsearch(Proteins, args.cpus, 1e-50, os.path.join(outputdir, 'annotate_misc'), pfam_results)
  File "/bigdata/bioinfo/pkgadmin/opt/linux/centos/7.x/x86_64/pkgs/funannotate/git-live/lib/library.py", line 965, in PFAMsearch
    with open(output, 'w') as out:
IOError: [Errno 2] No such file or directory: 'Results/annotate_misc/annotations.pfam.txt'
